### PR TITLE
Add Daemonette abilities and uniqueactions

### PR DESCRIPTION
**Pull request by HauntedSmores**
Been using this repo for a KT app and noticed these were missing.

### CODE REVIEW

**Comment by vjosset** [comment]
Funny timing - I noticed this myself yesterday when I was working on v2 of my app. Fixed but didn't push to github yet. I will double-check that I didn't miss anything in your pull and get it in. Thanks!

**Comment by vjosset** [comment]
Just out of curiosity, what are you doing with it? Something cool I imagine!

**Comment by vjosset** [comment]
Your commit matches what I had added, so I will push my changes in and close this pull request.
Note that the placeholder for triangle is "[TRI]", not "[TRIANGLE]" (annotated in comments on the commit).

File: compendium.json [hunk 1]
```diff
@@ -8222,8 +8242,29 @@
...
-                "abilities": [],
+                "uniqueactions": [
+                  {
+                    "factionid": "CHAOS",
+                    "killteamid": "DAEM",
+                    "fireteamid": "DETTE",
+                    "opid": "FTR",
+                    "uniqueactionid": "IC",
+                    "title": "Instrument of Chaos",
+                    "description": "Until the end of the Turning Point, each time a friendly DAEMONETTE operative performs a Normal Move or Charge action, it can move an additional [TRIANGLE].",
```
Comment (vjosset) [inline compendium.json:8253]: Placeholder for 1" (triangle) should be "[TRI]" instead of "[TRIANGLE]"

**Comment by HauntedSmores** [comment]
Awesome thank you @vjosset! Feel free to close when you push your updates.

**Comment by HauntedSmores** [comment]
> Just out of curiosity, what are you doing with it? Something cool I imagine!

Message me on Discord CreamSoda#3637!

**Comment by vjosset** [comment]
Main branch up to date

**Pull request closed without merging**